addons-l10n/ro/rounded-ui.json

```diff
@@ -0,0 +1 @@
+{"rounded-ui/@name":"Interfață cu colțuri rotunjite","rounded-ui/@description":"Face interfața site-ului Scratch mult mai plăcută, rotunjind colțurile unor obiecte."}
```

Comment:
You shouldn't add localized strings by yourself; those should be done through Transifex, after this PR is merged.

Reply:
Now I deleted the Romanian translation of my addon.